yaksh/file_utils.py

```diff
@@ -66,3 +67,31 @@ def is_csv(document):
     except (csv.Error, UnicodeDecodeError):
         return False, None
     return True, dialect
+
+
+def write_static_files_to_zip(zipfile, course_name, current_dir):
+    relative_folders = ["css", "js", "images"]
+    static_files = {"js": ["bootstrap.js", "bootstrap.min.js",
+                           "jquery-1.9.1.min.js", "video.js"],
+                    "css": ["bootstrap.css", "bootstrap.min.css",
+                            "video-js.css"],
+                    "images": ["yaksh_banner.png"]}
+    for folder in relative_folders:
+        folder_path = os.sep.join((current_dir, "static", "yaksh", folder))
+        for file in static_files[folder]:
+            file_path = os.sep.join((folder_path, file))
+            with open(file_path, "rb") as f:
+                zipfile.writestr(
+                    os.sep.join((course_name, "static", folder, file)),
+                    f.read()
+                    )
+
+
+def write_templates_to_zip(zipfile, template_path, data, filename, filepath):
```

Comment:
Cannot add static files in this function because static files will be added to the zip file only once. Whereas this function is called multiple times to add the different templates to the zip file.

Reply:
Can all templates be rendered first and then added with the static files?

Reply:
Just an idea that static argument default value is None.